module_utils/ansible_nailgun_cement.py

```diff
@@ -6,7 +6,7 @@
 import yaml
 
 from nailgun.config import ServerConfig
-import nailgun.entities
+from nailgun import entities
 
```

Comment:
Wow, you are fast with your responses :D

Yes, I totally agree, that sounds like the best solution :)